src/Spid/Saml/In/Response.php

```diff
@@ -76,8 +82,11 @@ public function validate($xml, $hasAssertion): bool
             )) {
                 throw new \Exception("Invalid IssueInstant attribute on Assertion");
             } elseif (strtotime($xml->getElementsByTagName('Assertion')->item(0)->getAttribute('IssueInstant')) >
-                strtotime('now') + $accepted_clock_skew_seconds) {
+                strtotime($root->getAttribute('IssueInstant'))) { // The issueInstant of assertion have to compared with issueInstant of the response, not with the current timestamp
                 throw new \Exception("IssueInstant attribute on Assertion is in the future");
+            } elseif (strtotime($xml->getElementsByTagName('Assertion')->item(0)->getAttribute('IssueInstant')) < // Check response 39 on demo validator
+                strtotime($root->getAttribute('IssueInstant'))) { // The issueInstant of assertion have to compared with issueInstant of the response, not with the current timestamp
```

Comment:
s/The issueInstant of assertion have to compared with issueInstant of the response, not with the current timestamp/The issueInstant of assertion have to be compared with the issueInstant of the response